synapse/models/gov/cn.py

```diff
@@ -20,8 +19,10 @@ def getBaseModels():
         name = 'gov:cn'
         return ((name, modl), )
 
-    @on('node:add', form='gov:cn:mucd')
-    def _onFormMucd(self, mesg):
+    def initCoreModule(self):
+        self.core.on('node:add', self._onAddMucd, form='gov:cn:mucd')
```

Comment:
core events are already distributed to coremodules.

Reply:
was thinking a bit about this...  having every loaded module link()'d to the event bus may eventually bite us.  It doesn't seem like too much of a burden to expect a module developer to register his callbacks directly with the cortex, but i'm biased ;)  Thoughts?

Reply:
We could also break out more "helpers" that implement the callback registration with the cortex....

Reply:
Under load we spend alot of time doing nothing in dist() for all the linked coremodules currently.  It just isn't super-evident; so moving towards a "more helpers" model which do linkages to the core directly is probably a good balance.

Reply:
thinking about this more - i think its reasonable to add more helpers so simple model callbacks don't need to think too hard; and everything else can require a dev to do explicit link  / onfini unlinking since they're doing something above and beyond simple model callbacks?